embed/web-extension/meson.build

```diff
@@ -10,6 +10,7 @@ web_extension_sources = [
 
 web_extension_deps = [
   ephymisc_dep,
+  ephysync_dep,
```

Comment:
Currently in lib/sync/meson.build you make a static library:

```
libephysync = static_library('ephysync',
  libephysync_sources,
  dependencies: libephysync_deps,
  include_directories: libephysync_includes
)
```

You want to make a shared library instead, and install it, otherwise the code will be duplicated in the web extension. Try this:

```
libephysync = shared_library('ephysync',
  libephysync_sources,
  dependencies: libephysync_deps,
  include_directories: libephysync_includes,
  install: true,
  install_dir: pkglibdir
)
```